trino/trino.go

```diff
@@ -249,6 +252,7 @@ func (c *Config) FormatDSN() (string, error) {
 		"session_properties": strings.Join(sessionkv, ","),
 		"extra_credentials":  strings.Join(credkv, ","),
 		"custom_client":      c.CustomClientName,
+		"bearer_auth":        c.BearerAuth,
```

Comment:
Can this be called `accessToken`, as in the JDBC driver? Also rename the config struct property.

Reference: https://trino.io/docs/current/client/jdbc.html#parameter-reference

Reply:
@nineinchnick - Fixed it. Please check.